tests/unit/metrics/test_replay_sched_lag_metrics.py

```
@@ -103,84 +130,109 @@ def test_parse_record_no_turn_timestamp_yields_no_value(self):
-        results = _derived_lag_results([0.0] * 5 + [float(tail_lag_ms)] * 5)
+        results = _inject([0.0] * 5 + [float(tail_lag_ms)] * 5)
+
+        assert results[ReplaySchedDegradedMetric.tag].avg == expected
+
+    def test_warn_degraded_called_once_with_percentiles(self) -> None:
+        calls: list[tuple[float, float, float]] = []
+        results = _inject([0.0] * 5 + [600.0] * 5, warn=lambda *a: calls.append(a))
+
+        assert results[ReplaySchedDegradedMetric.tag].avg == 1.0
+        assert len(calls) == 1
+        p50, p90, p99 = calls[0]
```

Ruff (0.15.21)

[warning] Unpacked variable `p50` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)